mmv1/templates/terraform/examples/dataform_repository.tf.erb

```diff
@@ -35,3 +35,25 @@ resource "google_dataform_repository" "<%= ctx[:primary_resource_id] %>" {
     table_prefix = "prefix_"
   }
 }
+
+resource "google_dataform_repository" "<%= ctx[:primary_resource_id] %>_git_ssh_connection" {
```

Comment:
I think this should get a separate example instead of being added here (ie. a new file like `dataform_repository_ssh.tf.erb`). That would make the examples more meaningful for users, and any test failures would be more clear.

Reply:
Sure